### PR TITLE
B12 padless

### PR DESCRIPTION
**Pull request by LynxSolstice**
# About the pull request

Adds a padless variant of the B12 Armor.

# Explain why it's good for the game

I wanted a variant of the B12 without shoulder padding so I made it an option.

# Testing Photographs and Procedure

Runs and compiles, shows up properly on-mob, in hand, and on snow maps.

# Changelog

:cl:LynxSolstice
add: Added the new variant of B12 armor to the appropriate vendors.
spellcheck: Added a new description for the B12 Padless Armor, changed the mention of "M3 B12" from the corpsman and engineer vendors.
imageadd: Cut the pads off the B12 armor for a subtype with no padding.
/:cl:

### CODE REVIEW

**Comment by DOC-surge** [comment]
I
 think this is a cool idea variation of some armors can be cool and this could be the first step to that.

**Review by fira — APPROVED** [review]
code OK

File: code/modules/clothing/suits/marine_armor.dm [hunk 1]
```diff
@@ -441,6 +441,11 @@ var/list/squad_colors_chat = list(rgb(230,125,125), rgb(255,230,80), rgb(255,150
 	armor_internaldamage = CLOTHING_ARMOR_MEDIUMHIGH
 	specialty = "B12 pattern marine"
 
+/obj/item/clothing/suit/storage/marine/leader/padless
+	name = "\improper B12 pattern padless marine armor"
+	desc = "A lightweight suit of carbon fiber body armor built for quick movement. Designed in a lovely forest green. This one is missing it's shoulder pads! Use it to toggle the built-in flashlight."
+	icon_state = "7_b"
+	specialty = "B12 pattern padless marine"
```
Comment (Zonespace27) [inline code/modules/clothing/suits/marine_armor.dm:448]:
```suggestion
	specialty = "B12 pattern padless marine"
	
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/marine_armor.dm (hunk 1 above)
Comment (HaultyAnonie) [reply code/modules/clothing/suits/marine_armor.dm:448]: what exactly did you suggest to change

**Comment by morrowwolf** [comment]
I don't really like the look of this.

**Pull request closed without merging**